spec/features/search_data_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
```

Comment:
Rename to search_users

Reply:
@VasylYaremkiv  is done --> https://github.com/CyberCraftInc/cyber-dashboard/pull/32/commits/f6c88fa1ddee8ef3878f1dc5931b8d03ad4f80ba